### ISSUE TITLE
Add SiFive

### ISSUE THREAD

**Issue by amalrkc** [gilbarbara/logos#681]
<div align="center">
<img src="https://upload.wikimedia.org/wikipedia/commons/e/ed/SiFive_Logo.png" width="70%">
</div>

**[SiFive](https://www.sifive.com/)**

SiFive, Inc. is a fabless semiconductor company and provider of commercial RISC-V processor IP and silicon chips based on the RISC-V instruction set architecture (ISA). SiFive's products include cores, SoCs, IPs, and development boards.

SiFive is one of the first companies to produce a chip that implements the RISC-V ISA. [Wikipedia](https://en.wikipedia.org/wiki/SiFive)

**Logo Code**

```
<svg xmlns="http://www.w3.org/2000/svg" width="100" height="34" fill="none" viewBox="0 0 100 34"><path fill="#252323" d="M4.274 17.6 8.862 3.375H26.44l1.784 5.534H12.696l-.992 3.14h17.533l2.636 8.175L17.65 30.708 4.135 20.739h9.127l4.349 3.207 8.605-6.343L4.274 17.6ZM28.56 0H6.742L0 21.016 17.651 34l17.651-12.988L28.56 0ZM53.086 9.614l-1.083 2.329c-1.68-1.03-3.358-1.456-4.529-1.456-1.524 0-2.519.582-2.519 1.635 0 3.425 8.352 1.59 8.33 7.232 0 2.8-2.43 4.523-5.833 4.523-2.43 0-4.728-1.008-6.319-2.485l1.127-2.284c1.59 1.478 3.579 2.284 5.236 2.284 1.811 0 2.894-.694 2.894-1.904 0-3.492-8.35-1.544-8.35-7.12 0-2.687 2.275-4.366 5.633-4.366 2.01 0 3.977.65 5.413 1.612M56.179 11.876h2.518v11.912h-2.518V11.876Zm2.74-3.404c0 .896-.642 1.545-1.481 1.545-.84 0-1.48-.649-1.48-1.545 0-.918.64-1.567 1.48-1.567s1.48.65 1.48 1.567ZM62.986 8.063l-.073.073v15.652l.073.073h.928l.072-.073V16.83h7.35l.073-.073v-.94l-.072-.074h-7.351V9.128h8.3l.073-.074v-.918l-.072-.073h-9.302ZM75.05 7.884c-.466 0-.845.394-.845.88 0 .497.38.901.845.901.467 0 .846-.404.846-.902 0-.485-.38-.88-.846-.88ZM74.586 12.049l-.072.073v11.666l.072.073h.906l.073-.073V12.122l-.073-.073h-.906ZM87.267 12.049l-.067.046-4.064 10.485-4.085-10.485-.068-.046h-.994l-.068.1 4.662 11.666.067.046h.95l.067-.046 4.64-11.666-.067-.1h-.973M89.976 17.31c.25-2.567 2.09-4.286 4.604-4.286 2.492 0 4.223 1.68 4.43 4.286h-9.034Zm10.023.903c.036-1.915-.533-3.577-1.6-4.679-.956-.987-2.276-1.508-3.819-1.508-3.293 0-5.683 2.498-5.683 5.94 0 3.429 2.39 5.917 5.683 5.917 1.807 0 3.411-.668 4.517-1.882v-.099l-.53-.604h-.108c-.922 1.024-2.284 1.588-3.834 1.588-2.643 0-4.513-1.845-4.674-4.6h9.976l.072-.073" style="--darkreader-inline-fill: #2d3031;" data-darkreader-inline-fill=""></path></svg>
```

**Comment by gilbarbara** [comment]
I'm not sure about this one. Risc-v, for sure, but this is just one of many companies building chips with the architecture, right?

**Comment by amalrkc** [comment]
Yes, you are correct but they play a much bigger role than just manufacture of RISC-V chips.

SiFive has made significant contributions to RISC-V research and development in several ways:

- Development of RISC-V ISA: SiFive was founded by the inventors of RISC-V, and they have been instrumental in the development of the RISC-V Instruction Set Architecture (ISA) since 2010.

- Commercialization of RISC-V: SiFive has played a key role in the commercialization of RISC-V, providing RISC-V solutions and having the largest investment and team dedicated to RISC-V.

- Innovation in Microarchitecture Design: SiFive’s innovation in microarchitecture design and configurability has led to an industry-leading portfolio of RISC-V processor cores for domain-specific SoC design. Their Core IP portfolio of RISC-V processor cores is based on unique microarchitecture generators that offer unmatched configurability.

- Bridging the Gap: While RISC-V is “free” and “open,” there is a significant gap between having RISC-V code and making a functioning chip. SiFive bridges this gap by providing the necessary add-ons, tools, and features needed to turn RISC-V code into a functioning chip.

- Leadership in the RISC-V Movement: SiFive has assumed a leadership role in the RISC-V movement, providing tested Intellectual Property (IP) and support for chip developers who incorporate RISC-V into their products. The company is leading the way for innovation on RISC-V and is playing a pivotal role in propelling the RISC-V industry into new frontiers of performance and applicability.

In summary, SiFive’s contributions to RISC-V research have been pivotal in the development, commercialization, and innovation of the RISC-V architecture. They continue to lead the way in advancing the RISC-V movement.

More information:

https://www.sifive.com/technology/risc-v

https://cambrian-ai.com/sifive-is-leading-the-way-for-innovation-on-risc-v/

https://www.sifive.com/press/sifive-partners-with-intel-to-spark-innovation-in-high-performance
